pkg/controller/flow/reconcile.go

```diff
@@ -266,26 +261,22 @@ func (r *reconciler) createSubscription(channelName string, target string, flow
 }
 
 func (r *reconciler) reconcileFeed(channelDNS string, flow *v1alpha1.Flow) (*feedsv1alpha1.Feed, error) {
-	feedName := flow.Name
-
-	feed := &feedsv1alpha1.Feed{}
-	err := r.client.Get(context.TODO(), client.ObjectKey{Namespace: flow.Namespace, Name: feedName}, feed)
+	feed, err := r.getFeedForFlow(flow)
```

Comment:
perhaps just `getFeed`  to match the `createFeed` method naming.

Reply:
Done.